var/spack/repos/builtin/packages/python-venv/package.py

```diff
@@ -93,6 +93,10 @@ def setup_dependent_run_environment(self, env, dependent_spec):
             path = os.path.join(dependent_spec.prefix, directory)
             if os.path.isdir(path):
                 env.prepend_path("PYTHONPATH", path)
+        env.prepend_path("PATH", dependent_spec.bindir)
```

Comment:
Looking at the CI failure, it looks like `bindir` is only defined for `python-venv` itself. Perhaps a definition should be added to `PythonExtension` in `build_systems`?

Reply:
Pushed, lets see how that goes.